src/components/core/collapsible-data-list-details.tsx

```diff
@@ -0,0 +1,42 @@
+import { PencilIcon } from "@heroicons/react/solid";
+import type { CollapsibleDataListEntry } from "./collapsible-data-list";
+
+type DetailsProps = {
+  hideEmpty?: boolean;
+  data: CollapsibleDataListEntry[];
+  readOnly?: boolean;
+  onEdit?: () => void;
+};
+
+export const Details = ({ data, hideEmpty = true, onEdit }: DetailsProps) => (
```

Comment:
Name should match file name.

```suggestion
export const CollapsibleDataListDetails = ({ data, hideEmpty = true, onEdit }: DetailsProps) => (
```